### ISSUE TITLE
Bullets don't render on last data point in a line series

### ISSUE THREAD

**Issue by newtonnthiga** [amcharts/amcharts5#637]
**Question**
I have an issue with bullets on line charts. When my line series has multiple data points, the bullets show up okay but the last bullet in the series is never rendered. Similarly, whenever there's only one data point in the series the bullet doesn't get rendered. 

I can't find anything about this behaviour in the docs.

In the chart below for example, notice how all other data points render a circle for the bullet, but the last point doesn't render anything. 
![Screenshot 2022-09-27 at 09 54 27](https://user-images.githubusercontent.com/101113464/192454878-ce205279-95ef-4495-969e-bb599a6ddde6.png)

xAxis
```
chart.xAxes.push(
    DateAxis.new(root, {
      max: new Date().getTime(),
      renderer: AxisRendererX.new(root, {
        minGridDistance: 60,
      }),
      baseInterval: {
        timeUnit: "month",
        count: 1,
      },
      tooltip: Tooltip.new(root, {}),
    })
  );
```

yAxis
```
let yAxis = chart.yAxes.push(
    ValueAxis.new(root, {
      renderer: AxisRendererY.new(root, {}),
      tooltip: Tooltip.new(root, {}),
    })
  );
```

series
```
// Example data
const data = [
 {
   "date":123456,
   "value":0.1,
 }
];

let series = chart.series.push(
      LineSeries.new(root, {
        name: displayName,
        xAxis: xAxis,
        yAxis: yAxis,
        showTooltipOn: "always",
        valueXField: "date",
        valueYField: "value",
        tooltip: Tooltip.new(root, {
          labelText: "{name}: {valueY.formatNumber('#.00')}%",
          background: Rectangle.new(root, {}),
          showTooltipOn: "always",
          pointerOrientation: "vertical",
          getFillFromSprite: false,
        }),
        legendLabelText: displayName,
        stroke: strokeColor,
      })
    );

series.bullets.push((root) => {
      return Bullet.new(root, {
        sprite: Circle.new(root, {
          radius: 3,
          fill: series.get("stroke"),
        }),
      });
    });

series.data.setAll(data);
```

**Codesandbox**
https://codesandbox.io/s/romantic-pateu-isucpo

**Environment (if applicable)**

 - amCharts version - `@amcharts/amcharts5@5.2.23`
 - Browser and its version - Google Chrome 
 - Related frameworks - `vue@3.2.38`

**Comment by martynasma** [comment]
I can't seem to be able to reproduce this.

Can you post your full chart (including data) on CodePen/jsFiddle/Sandbox/etc?

**Comment by newtonnthiga** [comment]
Yeah sure thing.

Here's the link
https://codesandbox.io/s/romantic-pateu-isucpo

**Comment by martynasma** [comment]
Fixed in [5.2.24](https://github.com/amcharts/amcharts5/releases/tag/5.2.24).

## [5.2.24] - 2022-09-27

### Added
- `lineType` (`"curved"`, `"straight"`) added to `MapLineSeries`. `"curved"` (default) connects points using shortest distance, which will result in curved lines based on map projection. `"straight"` connects points using visually straight lines, and will not cross the -180/180 longitude.
- `tooltipContainerBounds` added to `Root` settings. Allows defining margins around chart area for tooltips to go outside the chart itself.

### Changed
- If `tooltipLocation: 0` on a `DateAxis`, it will snap to the closest position now. Previously it snapped to the date over which mouse pointer was, this was not user-friendly in some setups. The same logic now applies to the `CategoryAxis` as well.

### Fixed
- WVAP indicator was showing ZigZag instead.
- Zoom-out button sometimes used to appear while initial animation was playing. Note: if `sequencedAnimation == true`, it will appear anyway.
- Fixed styles issue with date picker in `StockChart`.
- The last bullet of a series could become hidden if `baseInterval` was set to `"month"` on a related `DateAxis`.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
